lib/representable/logger.rb

```diff
@@ -0,0 +1,88 @@
+# Provides really basic logging functionality.
```

Comment:
I can easily refactor it to use https://ruby-doc.org/stdlib-2.1.0/libdoc/logger/rdoc/Logger.html

My main worry was not the logger but the behaviour. 
When people extend Representable::Debug they instantly get the desired output without messing with log levels or something. So I did this the default behaviour. And put at test_helper some code to disable it. But let the option to reenable it at the tests with an enviroment variable.

What do you suggest?